js-exercises/user-check-password/user.js

```diff
@@ -0,0 +1,19 @@
+
+// (*)
```

Comment:
We can remove the comments because they are basically guide to solve the problem.